### PR TITLE
Social net spinup

### PR DESCRIPTION
**Pull request by fukuru**

### CODE REVIEW

**Comment by andrewmarklloyd** [comment]
@fukuru once you fix the build issues we can merge into master and get it deployed

File: frontend/src/components/admin/forms/PodcastEditForm.tsx [hunk 1]
```diff
@@ -405,7 +456,7 @@ export default class PodcastEditForm extends React.Component<IPodcastEditFormPro
                                 })
                             :
                             <span>
-                                You don't have any collaborators left to remove
+                                You don"t have any collaborators left to remove
```
Comment (andrewmarklloyd) [inline frontend/src/components/admin/forms/PodcastEditForm.tsx]: Looks like a find and replace error

File: frontend/src/components/public/screens/PublicPodcastContactScreen.tsx [hunk 2]
```diff
@@ -60,76 +61,77 @@ export default class PublicPodcastContactScreen extends React.Component<IPublicP
...
     @autobind
-    public renderContactForm(){
+    public renderContactForm() {
         if (!this.props.publicClientState.podcast.email) {
             return null;
         }
 
         return (
             <Form>
-                <Form.Input required name="contactName" label='Name (required)' style={{maxWidth: "300px"}} value={this.state.contactName}
```
Comment (andrewmarklloyd) [inline frontend/src/components/public/screens/PublicPodcastContactScreen.tsx]: Not sure if these cause issues too

File: server/src/index.ts [hunk 3]
```diff
@@ -10,6 +10,6 @@ dotenv.config();
 import app from "./app";
 import { logger } from "./lib/logger";
 
-app.listen(config.port, () => {
-    logger.info(`App is running at port 3001`);
+app.listen(process.env.PORT, () => {
```
Comment (andrewmarklloyd) [inline server/src/index.ts]: Can we move this to the config module?

File: frontend/src/components/public/screens/PublicPodcastContactScreen.tsx (hunk 2 above)
Comment (fukuru) [reply frontend/src/components/public/screens/PublicPodcastContactScreen.tsx]: Doesn't appear to cause any issues

**Pull request closed without merging**